webapp/src/Entity/Clarification.php

```diff
@@ -118,7 +119,7 @@ class Clarification extends BaseApiEntity implements ExternalRelationshipEntityI
         nullable: false,
         options: ['comment' => 'Has been answered by jury?', 'default' => 0]
     )]
-    #[Serializer\Groups(['RestrictedNonstrict'])]
+    #[Serializer\Groups([AbstractRestController::GROUP_RESTRICTED_NONSTRICT])]
```

Comment:
Can we short the AbstractRestController to ARC to make this a bit shorter? I agree with constants but now there is a lot more reading needed. I think we can short GROUP also (but I get if you want that in the name..)

Reply:
I'm fine importing it as an `ARC` alias. I think we should keep the GROUP thing.